### PR TITLE
hotfix(staking):validators data in subquery

### PR DESCRIPTION
**Pull request by xsteadybcgo**
## Context

* [ED-1733](https://parallelfi.atlassian.net/browse/ED-1733?atlOrigin=eyJpIjoiN2NjZWEwYjFlYTViNGI5NDkxMTg5ZmZlMDIyYjBmMTAiLCJwIjoiaiJ9) 

## Were any new external packages added?

* No

### CODE REVIEW

**Comment by xsteadybcgo** [comment]
This PR I didn't use the existing Query class for the subquery as I didn't need the filter, the current class wrapper was tricky to modify and I coded a separate query, we do need to refactor this in the future.

File: packages/app/src/controller/staking/ledger.ts [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
   const data = await StakingLedger.find()
+  const validatorsData = await queryValidators(process.env.SUBQUERY_KUSAMA_ENDPOINT!)
+
   const res = await Promise.all(
     data.map(async ledger => {
-      const [unlockings, validators] = await Promise.all([
-        StakingLocking.find({ where: { ledgerId: ledger.id } }),
-        StakingValidator.find({ where: { ledgerId: ledger.id } })
-      ])
+      const unlockings = await StakingLocking.find({ where: { ledgerId: ledger.id } })
```
Comment (BruceChar) [inline packages/app/src/controller/staking/ledger.ts:13]: await in a map may occur some unexpected behavior

**Comment by BruceChar** [comment]
> This PR I didn't use the existing Query class for the subquery as I didn't need the filter, the current class wrapper was tricky to modify and I coded a separate query, we do need to refactor this in the future.

yes! this Query class is for subql pagenation, the common subql class is needed.

File: packages/app/src/models/heiko/staking/validators.ts [hunk 2]
```diff
@@ -1,25 +0,0 @@
...
-
-  @Column({ type: 'varchar' })
-  stakes!: string
-
-  @Column({ type: 'varchar' })
-  commission!: string
-
-  @Column({ type: 'int', name: 'block_number' })
-  blockHeight!: number
-}
```
Comment (BruceChar) [inline packages/app/src/models/heiko/staking/validators.ts:25]: don't we need this schema any more?

Comment (xsteadybcgo) [reply packages/app/src/models/heiko/staking/validators.ts:25]: We intend to use subquery without the migration-bot dependency, so this fix removes the old schema

**Review by BruceChar — APPROVED** [review]

**Pull request merged**